articles/azure/azs-how-create-service-principals.md

```diff
@@ -17,23 +17,21 @@ toc_mdlink: azs-how-create-service-principals.md
 
 ## Overview
 
-In Azure Stack you are able to grant an application access to stack resources by creating a service principal that uses Azure Resource Manager. A service principal lets you delegate specific permissions using **role-based access control**. As a best practice, you should use service principals for your applications. Service principals are preferable to running an application using your own credentials for the following reasons:
+In Azure Stack you are able to grant an external application access to Azure Stack resources by creating a service principal that uses Azure Resource Manager. A service principal lets you delegate specific permissions using **role-based access control**. Service principals are preferable to running an application using your own credentials for the following reasons:
 
-- You can assign permissions to the service principal that are different than your own account permissions. Typically, a service principal's permissions are restricted to exactly what the app needs to do.
+- You are able to assign specific permissions to the service principal that are different than your own account permissions. Typically, a service principal's permissions are restricted to exactly what the app needs to do which reduces the security risks involved with using external applications
```

Comment:
Needs a full stop at the end of the sentence.

Reply:
done